Orange/widgets/utils/state_summary.py

```diff
@@ -129,11 +131,11 @@ def b(s):
 
 def missing_values(value):
     if value:
-        return f'({value*100:.1f}% missing values)'
+        return f' ({value*100:.1f}% missing values)'
```

Comment:
I assume no one calls this externally, and moving a space inside won't be a problem. But I am curious why did you do so.

Reply:
Because of space-nazism. We had `f'Features: {features} {features_missing}'` where `features_missing` could be an empty string, resulting in a space at the end of the line. I wouldn't be able to sleep knowing that.